.eslintrc.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
   rules: {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
-    'prettier/prettier': ['warn', {}, { usePrettierrc: true }],
+    'prettier/prettier': ['warn', {}, { usePrettierrc: true, indent: 2 }],
```

Comment:
it's 2 by default, no?

Reply:
it's a change from https://github.com/epicmaxco/vuestic-admin/pull/982